include/rocksdb/env_encryption.h

```diff
@@ -23,9 +23,6 @@ struct ConfigOptions;
 // read from disk.
 Env* NewEncryptedEnv(Env* base_env,
                      const std::shared_ptr<EncryptionProvider>& provider);
-std::shared_ptr<FileSystem> NewEncryptedFS(
-    const std::shared_ptr<FileSystem>& base_fs,
-    const std::shared_ptr<EncryptionProvider>& provider);
```

Comment:
Why delete this?

Reply:
I saw it was only used in NewEncryptedEnv in .cc file, thought nobody will use it.
But consider it is in `include/*` directory, I've reverted this change.

Reply:
Thanks!